From 0fb1c949e421f6a30e197848d77a82570111d6af Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Tue, 25 Jun 2024 15:37:11 -0700
Subject: [PATCH] Docs/mm/damon/design: add a sysfs usage link for basic DAMON
 context setup

Readers of the design document's operations set layer section would
wonder how they can set the operations set and basic DAMON context
parameters.  Add a link to a section of DAMON sysfs interface usage
document that provides the answer on the design doc for easier browsing.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/mm/damon/design.rst | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Documentation/mm/damon/design.rst b/Documentation/mm/damon/design.rst
index 39e686c6862d..1ae9f2f43bec 100644
--- a/Documentation/mm/damon/design.rst
+++ b/Documentation/mm/damon/design.rst
@@ -71,6 +71,10 @@ describe how those work.
  - fvaddr: Monitor fixed virtual address ranges
  - paddr: Monitor the physical address space of the system
 
+To know how user-space can do the configuration via :ref:`DAMON sysfs interface
+<sysfs_interface>`, refer to :ref:`operations <sysfs_context>` file part of the
+documentation.
+
 
  .. _damon_design_vaddr_target_regions_construction:
 
-- 
2.39.2

